DQ-Loan-Xfer-Report/REPWRITERSPECS/BANNO.XFRTODQLOAN.RPT

```diff
@@ -0,0 +1,129 @@
+[*
+**  PowerOn Name:       BANNO.XFRTODQLOAN.RPT
+**
+**  Copyright 2020 Jack Henry and Associates
+**
+**  This Banno report PowerOn provides a listing of share>loan
+**  transfers where the loan is delinquent. This allows collections
+**  to review the accounts.
+**
+**  For more information check
+**  https://github.com/Banno/banno-powerons
+**
+**  Banno is not responsible for any modifications to this file
+**  made by unauthorized personnel.
+**
+**  DO NOT MODIFY THIS FILE UNLESS YOU KNOW WHAT YOU'RE DOING!
+*]
+
+TARGET=ACCOUNT
+
+DEFINE
+ FROMACCT=CHARACTER
+ TOACCT=CHARACTER
+ FROMID=CHARACTER
+ TOID=CHARACTER
+ XFRDATE=DATE
+ FROMDATE=DATE
+ TODATE=DATE
+ COUNT=NUMBER
+ SHAREOWNER=CHARACTER
+ LOANOWNER=CHARACTER
+END
+
+SETUP
+ FROMDATE=DATEREAD("Enter from date")
+ TODATE=DATEREAD("Enter to date")
+ COUNT=0
+END
+
+SELECT
+ ACCOUNT:CLOSEDATE='--/--/--' AND
+ ANY SHARE WITH (SHARE:CLOSEDATE='--/--/--' AND
+                 SHARE:CHARGEOFFDATE='--/--/--') AND
+ ANY SHARE TRANSFER WITH (SHARE TRANSFER:IDTYPE=1 AND
+                          SHARE TRANSFER:TYPE=3 AND
+                         (SHARE TRANSFER:EXPIRATIONDATE='--/--/--' OR
+                          SHARE TRANSFER:EXPIRATIONDATE>SYSTEMDATE)) 
+END
+
+PRINT TITLE="Share Transfers to DQ Loans" REPORTCATEGORY="COLLECTION"
+ HEADERS
+  COL=001 "ACCOUNT-FROM"
+  COL=019 "SHARE OWNER"
+  COL=037 "ACCOUNT-TO"
+  COL=056 "LOAN OWNER"
+  COL=080 "DUE DATE"
+  COL=094 "XFR DATE"
+  NEWLINE
+ END
+
+ SHAREOWNER=NAME:SHORTNAME
+
+ FOR EACH SHARE WITH (SHARE:CLOSEDATE='--/--/--' AND
+                      SHARE:CHARGEOFFDATE='--/--/--')
+  DO
+   IF ANY SHARE TRANSFER WITH (SHARE TRANSFER:IDTYPE=1 AND     [LOAN]
```

Comment:
Is this check necessary?  If there are no share transfers then the next code block will get skipped anyways.

Reply:
Done, removed IF statement